notebooks/Repo_Analysis.ipynb

```diff
@@ -33,6 +33,8 @@
    ],
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #15.                    !python -m srcopsmetrics.cli -clr $org/$repo -e Issue,PullRequest # noqa: E999
</pre><p>Is there a more python friendly way of doing this? can srcopsmetrics be imported ? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-services-group/metrics/pull/34/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>